voter/management/commands/voter_add_files.py

```diff
@@ -0,0 +1,51 @@
+import os
+import re
+from datetime import datetime
+
+from django.core.management import BaseCommand
+
+from voter.models import FileTracker, BadLine
```

Comment:
BadLine has gone away, so this will fail. What's the use case for this management command again?